tekton/utils/pipeline_run_builder.go

```diff
@@ -111,6 +111,20 @@ func (b *PipelineRunBuilder) WithObjectReferences(objects ...client.Object) *Pip
 	return b
 }
 
+// WithTenantArrayPrams constructs tektonv1.Param entries for each of the provided client.Objects.
+func (b *PipelineRunBuilder) WithTenantArrayParams(params []Param) *PipelineRunBuilder {
```

Comment:
Fine. I added this because it was not working passing the parameters to WithObjectReferences just like the managed one. I will figure out how to fix it.

Reply:
As Johnny said, in this case you can (and should) use WithParams directly.

Reply:
Ok. Please take a look if this is done correctly now.